src/link_renderer.ts

```diff
@@ -120,7 +119,7 @@ export class LinksRenderer {
       var $combi = $group.selectAll('g').data(combinations);
       $combi.enter().append('g');
       $combi.exit().remove();
-      $combi.attr('data-id', identity);
+      $combi.attr('data-id', String);
```

Comment:
Ok, good to know. Even though it's hard to understand when you read it for the first time.